packages/cmf/src/Dispatcher.js

```diff
@@ -89,10 +60,19 @@ export class Dispatcher extends React.Component {
 		}
 	}
 
+	checkIfActionInfoExist() {
+		action.getOnProps(this.props).forEach(name => {
+			if (typeof this.props[name] === 'string') {
+				actionCreator.get(this.context, this.props[name]);
+			}
+		});
+	}
+
 	/**
 	 * @return {object} ReactElement
 	 */
 	render() {
+		this.checkIfActionInfoExist();
```

Comment:
AFAIK if you don t bind it it should not have access to this.props and this.context

Reply:
This is a method call, not a function call. So this is ok, it has the context